js/Numbers/1-celcius-farenheit.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var convertCToF = function(celcius){
```

Comment:
Don't change this, unless I make other comments - but better naming would be needed in live code:
````
var convertCelciusToFahrenheit = .......
````

The JS team will be this strict.